### ISSUE TITLE
Belief in God Topic

### ISSUE THREAD

**Issue by HowAreWeAll** [Monika-After-Story/MonikaModDev#5993]
I know that Monika will ask about belief in a God, however, the choice she gives is a "Yes" or "No". I believe it would be a good option if we could say we are "Unsure" or say "I Don't Know", as there are people who don't know if they believe in God/Gods, or are Agnostic, or something like that.

**Comment by Freshairkaboom** [comment]
Being agnostic isn't about belief though, the definition of agnosticism is: "A person who believes that nothing is known or can be known of the existence or nature of God." 

This is popularly called philosophical agnosticism, and is the "strong" position that an agnostic can take, the weak one being personal ignorance about the knowledge of God's existence, also known as the colloquial definition of agnostic. It's fully possible to be both an atheist and a theist and still be an agnostic.

Believing in god or not is a binary proposition. Not knowing is simply not answering the question, it's not a position in and of itself. I can see where the confusion begins though, as philosophical atheism is often regarded as the active belief that there are no gods, while the colloquial term actually allows for agnosticism to co-exist with it. 

So when Monika asks whether or not you believe in god, I think there should be a third option that is something like "let me think about it" or "I don't want to discuss it", for those that haven't really considered it.

However, definitionally, if you don't know whether or not you believe in god, at that particular moment, you are in fact an atheist, and should answer no. I think I can speak for most theologians when I claim that theism requires active belief in a deity, like you have to mentally be participating in the believing of it, otherwise you're just pretending to be a theist.

If you don't believe me, that's fine, I'm not here to debate, just to tell you what I know on the subject. It's a sore topic, and not many people are comfortable calling themselves atheist (non-theist is somehow a more comfortable term though). I hope I'm not stepping on any toes here, as I'm trying to be as unbiased as possible.

**Comment by multimokia** [comment]
Technically agnosticism is a fair stance. Basically saying you don't believe there *is* or is *not* a god. Which in fact is valid due to the burden of proof argument. Of course, there's a branch I'm aware of, that being 'agnostic-atheist`. Which is basically leaning more toward the side of atheism (living as if there isn't a god, but still not believing one way or the other)

Not knowing, while true, it doesn't exactly answer the question. It is also the logical stance to take as nobody can prove one side or the other. By not picking a definite side, you admit that you yourself do not have enough information to make an argument for one or the other.

**Comment by Freshairkaboom** [comment]
It is a fair stance about a different question, not the belief in god question :P

**Comment by multimokia** [comment]
Well, it's a valid statement. It's fair game to say you don't know, agnostics cannot claim yes or no.

**Comment by Freshairkaboom** [comment]
True I'm just making sure we're not stealing definitions from like 80% of self-proclaimed atheists here and giving them all an unnecessary burden of proof.

**Comment by multimokia** [comment]
That's two different sides of a coin though. One has definite proof for whether or not you do play or not. The argument in terms of there being a god isn't black and white purely because there's no proof for either side.

**Comment by Freshairkaboom** [comment]
Belief doesn't require proof. Belief is an activity that you either participate in, or you don't. It has nothing to do with the strength of the evidence for or against the proposition, and belief itself need not be justified at all.

**Comment by multimokia** [comment]
But that's the point of agnosticism. You literally do not believe any which way, purely due to the burden of proof argument

**Comment by Necrocytus** [comment]
Just trying to bring a couple different points of view here. I think OP's suggestion or something similar to it would be valid since other than being religious, agnostic, or atheist as you all mention there are many other different set of beliefs that go slightly beyond the concepts already discussed, for example patheism or apatheism. What I'm trying to say is that there are ideologies and beliefs in which the very existance universe can be defined as "god" to someone, or maybe the existance/non-existance of god is irrelevant to the player. Maybe that question is actually really vague to begin with since the scope of the answers is limited to some mindsets.

**Comment by Freshairkaboom** [comment]
I also think it's valid and probably should be part of it. I'm not discussing OP's suggestion, so I derailed a little yesterday. I think it's a great idea what he's proposing, I just made sure we were on the same page as to why. I don't support this because I think agnosticism is some middle ground on the question of belief in god, but rather exists on a spectrum alongside the binary proposition of whether or not you believe in god.

As you said, pantheism believes in a god that is everything in the universe, so they would answer yes, while an apatheist would answer no because participating in a belief in god doesn't matter to them, a key trait non-theists have.

In any case, I think we should discuss this further on the Discord if you want, I agree with OP's proposition, and I'd like to see it being done with respect for everyone's views.
